builder/oracle/bmcs/artifact.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2017 Oracle America, Inc.
```

Comment:
Would you be able to remove the copyright notices? It's our policy not to have these in the source.

Reply:
I'll chase this up today and get back to you. TBH not sure. Would it be a blocker to release in it's current form?